.github/scripts/delete-webhook-requests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Script used to delete the WebHook site requests which are older than 24 hours
+
+webhook_base_path=https://webhook.site
+token=$1
+end_date=$(date "--date=now -1day" +"%Y-%m-%d%%20%H:%M:%S")
+
+if [[ -z "$token" ]]
+then
+    echo "Mandatory token parameter not provided"
+    exit 1
+fi
+
+uuids=$(curl --location $webhook_base_path/token/$token/requests?date_to=$end_date | jq '.data[].uuid' | tr -d \")
+
+curl_status=$?
+if [ $curl_status -ne 0 ]; then
+  exit $curl_status
+fi
+
+for uuid in $uuids
+do
+  curl --location -S -s -D /dev/stderr -XDELETE $webhook_base_path/token/$token/request/$uuid
+
+  curl_status=$?
+  if [ $curl_status -eq 0 ]; then
+    echo "Deleted UUID $uuid"
+  else
+    exit $curl_status
```

Comment:
Added log message.
IMHO adding some logic which would try to delete all the requests and check whether any of the delete requests failed would be unnecessary overkill.

Reply:
Actually I didn't want to check whether any of the request failed, but just simply continue and try to delete all of the remaining request rather than exiting the script. I was just thinking about something simpler below, and just try to delete as many request as possible:

```suggestion
    for uuid in $uuids
    do
      curl --location -S -s -D /dev/stderr -XDELETE $webhook_base_path/token/$token/request/$uuid
    
      curl_status=$?
      if [ $curl_status -eq 0 ]; then
        echo "Deleted UUID $uuid"
      else
        echo "Failed to delete Webhook request $uuid"
      fi
    done
```

But don't have any strong opinion on this. I'm fine with this

Reply:
The point is that this approach will result in a green successful GHA execution even when there are issues deleting the requests. So it will make impossible to notice that something is broken without investigating the logs.

Reply:
got it! you're totally right. Thanks for the explanation!